### DISCUSSION TITLE
RetroScaler Blueretro PS4 dualshock

### DISCUSSION THREAD

**Discussion by duna81** [darthcloud/BlueRetro#859] [Q&A]
i received my bluetooth adapter today. First i installed latest firmware 1.9.1 with BlueRetro OTA FW update. Then i pluged it to my ps2. i can sync blueretro with my ps4 dualshock. i have ps2 with (ide hdd with free mcboot and also has chip for backup cd-r (dont know chip brand). nice connection (steady lights on both sides) but nothing happening. dualshock doesnt work at menu or in game. when i connect ps2 controller to second port it is working. i tried bleuretro at controller 1 and 2. nothing changed.

i want to try adapter at psone. it instantly connects dualshock. at menu and in game i can use it flawless. even in gran turismo 1 i can use it without any problem. 

any one have an idea whats going on. why it isnt working with my ps2 :( 

thank you all.

**Comment by duna81** [comment]
update:

Same setup works with PS1. i can use it with my chiped PS ONE and PS1

On the other hand i tried with ps2 no mod chip, ps2 slim chiped and no mod chiped. non of them worked. i cant use the same setup with ps2. and i used it with out free mcboot. same... not working. so it is not about mod chip or free mcboot. 

any ideas?

**Comment by darthcloud** [comment]
What are the model numbers of your PS2?

Can you post pictures of your adapter connected to the PS2s?
